runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -1115,7 +1120,7 @@ class CompilationInfo
    bool                   _traceCompiling;
 #endif
    int32_t                _compThreadIndex;
-   int32_t                _numDiagnosticThreads;
+   int32_t                _diagnosticThreadIndex;
```

Comment:
Are `_diagnosticThreadIndex` and `_compThreadIndex` supposed to be 0-based indices or a count of the number of threads? I see it them being used as both and it is a little confusing.

Reply:
@ymanton Looks like `_diagnosticThreadIndex` and `_compThreadIndex` are used more as the number of the threads instead of as an index. Should I change them to `_numDiagnosticThreads` and `_numCompThreads`?